modules/modes/SelectMode.js

```diff
@@ -157,6 +158,7 @@ export class SelectMode extends AbstractMode {
 
     } else if (datum instanceof QAItem && datum.service === 'maproulette') {
       sidebarContent = uiMapRouletteEditor(context).error(datum);
+      uiMapRouletteMenu(context).error(datum);
```

Comment:
```suggestion
      let uiSystem = this.context.systems.ui;
      uiSystem.MapRouletteMenu.error(datum);
```

Reply:
You've already instantiated a new uiMapRouletteMenu object and attached it to the UiSystem via the `MapRouletteMenu` variable- this call completely re-creates the object, so selectMode and UiSystem aren't able to sync up. 

With this change, you should see the _qaItem variable be defined as you take a spin through the body of the menu rendering code.